src/plugins/data_source_management/public/components/create_data_source_wizard/components/create_new_credential/create_new_credential.tsx

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
if we have to have this embedded way to create data source, is it better to have a shared widget of credential creation between this page and the credential management page?